tasks/install.debian.yml

```diff
@@ -19,21 +19,22 @@
 
 - name: Fail when used wrong mongodb_version variable with Debian Stretch
   fail:
-    msg: "mongodb_version variable should be '3.2' or '3.6' for Debian Stretch"
+    msg: "mongodb_version variable should be '3.2' or '3.6' or '4.0' for Debian Stretch"
```

Comment:
BTW 3.2 reached EOL last September. If you can get rid of it in this PR, it will be very appreciated.

Reply:
Done!